modules/460-log-shipper/images/vector/main.go

```diff
@@ -92,11 +90,17 @@ func main() {
 	<-make(chan struct{})
 }
 
-func reloadVectorConfig() error {
+func reloadVectorConfig() (err error) {
 	tempConfigDir, err := os.MkdirTemp("", "vector-config")
```

Comment:
It should be fixed, I pointed to the problem when we were discussing it internally. 
Files are created in the tmp dir and deleted only after the pod restart.

Reply:
It is fixed by introducing a defer with cleanup.